test/constants.ts

```diff
@@ -294,5 +305,60 @@ export const DUTCH_LIMIT_ORDER_JSON = {
+    input: {
+      token: '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984',
+      startAmount: AMOUNT,
+      endAmount: AMOUNT,
+    },
+    outputs: [
+      {
+        token: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+        startAmount: '10000',
+        endAmount: '9950',
+        recipient: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+      },
+      {
+        token: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+        startAmount: BigNumber.from('10000').mul(PORTION_BIPS).div(BPS).toString(),
+        endAmount: BigNumber.from('9950').mul(PORTION_BIPS).div(BPS).toString(),
+        recipient: PORTION_RECIPIENT,
+      },
+    ],
+  },
+  encodedOrder:
+    '0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003c0000000000000000000000001111111111111111111111111111111111111111000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000001f9840a85d5af5bf1d1762f925bdaddc4201f9840000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000de0b6b3a764000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000006000da47483062a0d734ba3dc7576ce6a0b645c4000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000048000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000000000000000000000000000000000000000000000000000000000000271000000000000000000000000000000000000000000000000000000000000026de000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000d8da6bf26964af9d7eed9e03e53415d37aa96045',
+  quoteId: 'quoteId',
+  requestId: 'requestId',
+  startTimeBufferSecs: 45,
+  auctionPeriodSecs: 60,
+  slippageTolerance: '0.5',
+  orderHash: '0x0ce5181c85484b2c506800056a75ad25e0eea224fb067071447536905b8cf3b3',
 };
+
+export const GREENLIST_TOKEN_PAIRS: Array<[Currency, Currency]> = [
+  [Ether.onChain(ChainId.MAINNET), USDC_ON(ChainId.MAINNET)],
+  [WETH9[ChainId.MAINNET], USDT_ON(ChainId.MAINNET)],
+  [DAI_ON(ChainId.MAINNET), WBTC_MAINNET],
+];
+
+export const GREENLIST_CARVEOUT_PAIRS: Array<[Currency, Currency]> = [
+  [USDC_MAINNET, DAI_MAINNET],
+  [WETH9[ChainId.MAINNET], Ether.onChain(ChainId.MAINNET)],
+];
```

Comment:
weird, this should be added in https://github.com/Uniswap/unified-routing-api/pull/278/files#diff-8026d4e3c93ec8df13daa287704254d06f0245d5f6901bb7073e0b8eb41b3222R303-R312 right?

Does hurt, but maybe check your merge?